redisbp/hooks.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
 	"errors"
 	"fmt"
-
```

Comment:
And don't remove this blank line. See: https://github.com/reddit/baseplate.go/blob/master/Style.md#import-groups